compiler/p/codegen/OMRCodeGenerator.cpp

```diff
@@ -1831,7 +1831,8 @@ bool OMR::Power::CodeGenerator::getSupportsOpCodeForAutoSIMD(TR::CPU *cpu, TR::I
       case TR::mloadi:
       case TR::mstore:
       case TR::mstorei:
-         if (et == TR::Int32 || et == TR::Int64 || et == TR::Float || et == TR::Double)
+      // since lxvb16x, stxvb16x, lxvh8x, and stxvh8x are not available on P8 and lower, vector/loads and stores should only be enabled under these conditions:
```

Comment:
Please add 3 more spaces before `//`.
This comment line and the following `if` line should have the same indentation.

Reply:
Fixed!